.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
```

Comment:
Just curious, was there a version 1?

Reply:
Github mandates the version to be 2
<img width="726" alt="image" src="https://user-images.githubusercontent.com/102190347/210611136-6108665a-0929-4605-8721-7ecc6d309d9b.png">

Reply:
fascinating